client/tests/get_status_test.py

```diff
@@ -0,0 +1,155 @@
+import pytest
+import featureform as ff
+from featureform.resources import TrainingSet, Feature, ResourceColumnMapping
+from featureform.serving import ServingClient
+
+from datetime import timedelta
+
+from .proto import metadata_pb2_grpc, metadata_pb2
+
+def test_pending_training_set_hosted(mocker):
```

Comment:
make your life easier and use @pytest.mark.parametrize

Reply:
Made it paramaterized